openedx_proversity_reports/reports/enrollment_report.py

```diff
@@ -0,0 +1,82 @@
+"""
+Enrollment report Report Class.
+"""
+import logging
+from datetime import datetime
+
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+
+from openedx_proversity_reports.edxapp_wrapper.get_student_account_library import get_user_salesforce_contact_id
+from openedx_proversity_reports.edxapp_wrapper.get_student_library import get_course_enrollment, get_user_profile
+
+
+LOG = logging.getLogger(__name__)
+
+
+class EnrollmentReport(object):
+    """
+    EnrollmentReport Class.
+    """
+
+    def __init__(self, course_id):
+        try:
+            self.course_key = CourseKey.from_string(course_id)
+        except InvalidKeyError:
+            LOG.error('Invalid course_id = %s for learning tracker report', course_id)
+            raise InvalidKeyError
+
+    def generate_report(self, **kwargs):
+        """
+        Returns a List with the enrollments for the given dates.
+        """
+        updated_at = kwargs.get('updated_at', '')
```

Comment:
We are not sanitizing these variables, so maybe we need a Serializer, or something to sanitize these values. @andrey-canon